zimui/src/plugins/vuetify.ts

```diff
@@ -9,9 +10,9 @@ async function loadVuetify() {
 
   // Load primary and secondary colors from config.json
   try {
-    const response = await fetch('./config.json')
-    if (response.ok) {
-      const config: Config = await response.json()
+    const response = await axios.get('./config.json')
+    if (response.status === 200) {
```

Comment:
Replace `200` by `HttpStatusCode.Ok`, with `import { HttpStatusCode } from axios`

Reply:
Changed in [664875a](https://github.com/openzim/youtube/pull/272/commits/664875ab24710957d5240d3a12f786544b72ad70)